README.md

```diff
@@ -4,6 +4,14 @@
 1. Create an image configuration by specifying things like release code, packages, scripts to run after building, and filesystem overlays.
 2. Build the image.
 
+## Pre-built Images
+Pre-built qcow2 images are available as [Packages](https://github.com/orgs/sandialabs/packages?repo_name=sceptre-phenix-images). They are stored in OCI registry format (due to size constraints) and require the [oras](https://oras.land/docs/installation) client to download. Once you have the oras client installed you can download with a command like:
+
```

Comment:
Perhaps have a note here that it will result in `bennu.qc2` in current working directory. I suspected that was the case, but had to go digging in ORAS docs to be sure.